integration_test/deploy-dr-buckets.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -ex
+
+export AWS_ACCESS_KEY_ID=$bamboo_AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$bamboo_AWS_SECRET_ACCESS_KEY
+export AWS_DEFAULT_REGION=$bamboo_AWS_DEFAULT_REGION
+
+#remove old files from bamboo as they throw error
+rm *.tf
+git clone --branch develop --single-branch https://github.com/nasa/cumulus-orca.git && cd integration_test
+#replace prefix with bamboo prefix variable
+sed -e 's/PREFIX/'"$bamboo_PREFIX"'/g' dr-buckets.tf.template > dr-buckets.tf
+
+if ! aws s3 cp s3://$bamboo_PREFIX-dr-tf-state/dr-buckets-tf/terraform.tfstate .;then
```

Comment:
Are we checking for a file here or the bucket, or both? Not sure we even really need this check since with terraform we should be able to run over and over again.

Reply:
we are checking for the terraform.tf state file. 

When a user runs the pipeline for the first time, the script will fail without this check because there is no .tfstate file or prefix-tf-state bucket. So if it cannot copy the state file, then it means the prefix-tf-state bucket is not created and will deploy all other buckets.

Next time the user reruns the pipeline, this check will find the .tfstate file and only update if there is a modification in terraform resource.

Reply:
In the first check, if it's not able to copy the .tfstate file from the s3 bucket, it means the file is not present and the bucket is not created. If the check is able to copy the tfstate file, it means the bucket is already created and then it configures the s3 backend. 

If we want to manually create the bucket for tf state first, then configuring the backend will be easier.

Reply:
Lets talk through this during orcatechture or after standup tomorrow.